src/constants/index.tsx

```diff
@@ -155,6 +155,12 @@ export const BLOCK_DIFFERENCE_THRESHOLD = {
   [SupportedNetwork.ARBITRUM_ONE]: 200, // Arbitrum one has multiple blocks in the same second
 };
 
+export const FACTORY_STARTING_BLOCK = {
```

Comment:
would these values change as time moves on? What's the criteria to come up with these values?

Reply:
These values won't change, they are the blocks at which the contracts were deployed.